src/commcare_cloud/terraform/modules/commcarehq/main.tf

```diff
@@ -78,7 +78,7 @@ resource "aws_eip" "proxy" {
 module "Redis" {
   source               = "../elasticache"
   create               = "${lookup(local.redis, "create", true)}"
-  cluster_id           = "${var.environment}-redis"
+  cluster_id           = "redis-${var.environment}"
```

Comment:
how come this is changing?

Reply:
I noticed I had named this backwards from how other resources are named, so I decided to change it to match. I noticed this when I was adding the code to look up the redis endpoint directly from AWS.